### ISSUE TITLE
Disabled select does not communicate disabled state to screen reader with using VoiceOver on iOS

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#1906]
# Bug Report

## Description
When interacting with disabled select, VoiceOver doesn't provide any communication to the user that the select is disabled. VoiceOver on iOS will usually say "Dimmed" after interacting with an input that is disabled.

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-core/#/tests/terra-form-select/form-select/disabled
2. Interact with Select component
3. VO will read, "Select a color" without indicated the input is disabled. Though user is unable to select a color.

## Expected Behavior
If select is disabled, it should communicate that information to screen reader.

## Possible Solution
Likely need to take a look at the markup structure and rework some part of it.

## Environment

* Component Name and Version: terra-select
* Browser Name and Version: Mobile Safari 12 on iOS w/ VoiceOver
